packages/react-core/src/components/MenuToggle/examples/MenuToggle.md

```diff
@@ -215,19 +215,19 @@ Variant styling can be applied to split button toggles to adjust their appearanc
 
 ### Split toggle with labeled checkbox
 
-To add a text label to a split button toggle, pass `children` to the `<MenuToggle>` component.
+You can allow users to select a toggle checkbox by clicking either the checkbox or the text label.
 
-```ts file='MenuToggleSplitButtonCheckboxWithToggleText.tsx'
+To do so, pass `children` to the `<MenuToggleCheckbox>` component. When the menu toggle text is clicked, the checkbox's `onChange` callback will be triggered.
 
-```
+```ts file='MenuToggleSplitButtonCheckboxWithText.tsx'
 
-### Split toggle with checkbox and toggle text label
+```
 
-You can allow users to select a toggle checkbox by clicking either the checkbox or the text label.
+### Split toggle with checkbox and toggle text 
 
-To do so, pass `children` to the `<MenuToggleCheckbox>` component. When the menu toggle text is clicked, the checkbox's `onChange` callback will be triggered.
+To add a text label to a split button toggle, pass `children` to the `<MenuToggle>` component.
```

Comment:
```suggestion
To link a split toggle label to the toggle button itself, pass the label to the `<MenuToggle>` component, instead of `splitButtonOptions`.
```